### PR TITLE
includes : disable static checks when using polymorphic relations

### PR DESCRIPTION
**Pull request by ThomasSevestre**
when including `whatever.polymorphic_relation.something_else`, original code complains about `something_else` not being compatible with `whatever`

### CODE REVIEW

**Comment by stas** [comment]
@ThomasSevestre would you be kind to provide a test case to replicate this scenario pretty please?

**Comment by ThomasSevestre** [comment]
any feedback ?

File: spec/lib/object_serializer_polymorphic_spec.rb [hunk 1]
```diff
@@ -48,4 +75,13 @@ class ListSerializer
       expect(record_type).to eq 'car'.to_sym
     end
   end
+
+  context 'when serializing included polymorphic relationships' do
+    it "should not raise" do
+      list = List.new
+      list.id = 1
+      list.items = [checklist_item, car]
+      expect(-> {ListSerializer.new(list, include: ["items.list"])}).not_to raise_error
```
Comment (stas) [inline spec/lib/object_serializer_polymorphic_spec.rb]: RSpec syntax allows passing directly a block:
https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher

**Comment by christophersansone** [comment]
Just an FYI, I saw this as well when building #32 and fixed it there too.  Same logic, just `break` if the serializer may change.

**Comment by stas** [comment]
Superseded by #32

**Comment by stas** [comment]
Thanks again @ThomasSevestre

**Pull request closed without merging**